bedevere/stage.py

```diff
@@ -197,6 +206,27 @@ async def new_review(event, gh, *args, **kwargs):
             raise ValueError(f"unexpected review state: {state!r}")
 
 
+@router.register("pull_request_review", action="dismissed")
```

Comment:
@ezio-melotti In response to your comment, this is the only new registered route.

However, iterating on reviewers and core_reviewers might also be requests? I'm not sure which requests are counted towards the limit.